.github/workflows/ci.yml

```diff
@@ -33,9 +33,6 @@ jobs:
     - name: Check Licenses
       run: node Makefile checkLicenses
 
-    - name: Lint Docs JS Files
```

Comment:
We wanted this a separate step so we could more easily tell where a failure was on a PR.

Reply:
The GitHub action from trunk will annotate the files inline with the issue. Is that not sufficient to get an understanding of the source of the problem. 

I can create a sub branch to demonstrate

Reply:
Will those show up on the Discussion tab of the PR? What I'm looking for is the "glance-ability" of a PR so I don't have to switch to the code tab to get a general understanding which CI checks failed. This is key when we are barraged with PRs and trying to triage.

Reply:
ping @EliSchleifer